tls/s2n_connection.c

```diff
@@ -909,11 +910,28 @@ int s2n_connection_set_read_fd(struct s2n_connection *conn, int rfd)
     return 0;
 }
 
+int s2n_connection_get_read_fd(struct s2n_connection *conn, int *readFd)
+{
+    POSIX_ENSURE_REF(conn);
+    POSIX_ENSURE_REF(readFd);
+
+    /* in case we do not use managed io, or in case read fd was never set we should return invalid fd */
+    if(conn->managed_io == 1 && conn->recv_io_context) {
```

Comment:
```suggestion
    /* in case we do not use managed io, or in case read fd was never set we should return invalid fd */
    if (conn->managed_io == 1 && conn->recv_io_context) {
```